errors/go.mod

```diff
@@ -1 +1,3 @@
 module github.com/decred/dcrwallet/errors
+
+go 1.12
```

Comment:
Can you remove these changes from the unrelated modules?  PRs should be focused on only changing one thing at a time.

Reply:
The changes have been removed from the unrelated modules, and CI successful...